generate_nor.c

```diff
@@ -330,7 +368,7 @@ int main(int argc, char *argv[]) {
 
     // create "common" partition with env variables
     chrp_nvram_header *partition_header = (chrp_nvram_header *)(nvram_data + sizeof(apple_nvram_header));
-    char *env = "boot-args=debug=0x8 kextlog=0xfff cpus=1 rd=disk0s1 serial=1 io=0xffff8fff";
+    char *env = "debug-uarts=3";
```

Comment:
I'm pretty sure that the boot args in the NOR are useless for the IT2G so I think this line can be removed?

Reply:
Yes that can be removed. I have noticed however that boot-args="" appears in the actual NOR of the device. (It is empty however)